GVFS/GVFS.FunctionalTests/Tests/FastFetchTests.cs

```diff
@@ -491,9 +491,13 @@ private string RunFastFetch(string args)
             processInfo.RedirectStandardError = true;
             
             ProcessResult result = ProcessHelper.Run(processInfo);
-            result.Output.Contains("Error").ShouldEqual(false, result.Output);
-            result.Errors.ShouldBeEmpty(result.Errors);
-            result.ExitCode.ShouldEqual(0);
+            result.Output.Contains("Error").ShouldEqual(expectError, result.Output);
+            if (!expectError)
+            {
+                result.Errors.ShouldBeEmpty(result.Errors);
+                result.ExitCode.ShouldEqual(0);

```

Comment:
If `expectError` is `true` can\should we verify that `ExitCode` is non-zero?

Reply:
Oh yep, thanks